docs/containers-storage.conf.5.md

```diff
@@ -174,6 +174,9 @@ The `storage.options.overlay` table supports the following options:
 **ignore_chown_errors** = "false"
   ignore_chown_errors can be set to allow a non privileged user running with a  single UID within a user namespace to run containers. The user can pull and use any image even those with multiple uids.  Note multiple UIDs will be squashed down to the default uid in the container.  These images will have no separation between the users in the container. (default: false)
 
+**inodes**=""
+  Maximum inodes of a container image.   This flag can be used to set quota on the inodes of container images. (format: <number>)
```

Comment:
Perhaps?
```suggestion
  Maximum inodes of a container image.   This flag can be used to set a quota on the inodes allocated for a container image. (format: <number>)
```
or "inodes that can be used by a container image."

Reply:
Both are a bit misleading: because we only apply quota limits to read-write layers, images are not affected by quotas.